### PR TITLE
Extend get_events bugfix according to the previously implemented tests

### PR DESCRIPTION
**Pull request by eozd**
Changes proposed in this pull request:
- In #15 we fixed the issue regarding missing the first marker in a file. However, our implementation misses a few cases that were tested in `pspm_get_events_test`. This pull request handles these issues. The main idea of the proposed changes are as below:
  * We missed two usecases: the input signal may be shifted by a baseline and the pulses might also be triangular. To handle both of these cases with the new code, there is a new signal normalization portion added.
  * Since we use a normalized signal for detection, the original signal is stored in an extra variable in order to return correct marker values later in the code.

### CODE REVIEW

File: src/pspm_get_events.m [hunk 1]
```diff
@@ -108,7 +115,7 @@
         baseline = v(end, 1);
         
         % we are interested in the delta -> remove "baseline offset"
-        values = data(round(mPos)) - baseline;
+        values = data_orig(round(mPos) - 3) - baseline;
```
Comment (eozd) [inline src/pspm_get_events.m]: I am not really happy by this `-3` part. Maybe there is a better way to implement it?

Comment (irojkov-ph) [reply src/pspm_get_events.m]: What this 3 is doing here ? Why do we need it ?

**Comment by eozd** [comment]
> What this 3 is doing here ? Why do we need it ?

So `mPos` is defined using `data` which has 3 extra zeroes padded at the front. Therefore, its indices will be offset by 3

**Comment by irojkov-ph** [comment]
Ok I see, fair enough. For now I think it is enough and for 4.3.0 we will anyway modify this function.

**Pull request merged**